beacon-chain/blockchain/process_block.go

```diff
@@ -482,52 +483,6 @@ func (s *Service) onBlockBatch(ctx context.Context, blks []interfaces.ReadOnlySi
 	return s.saveHeadNoDB(ctx, lastB, lastBR, preState)
 }
 
-// Epoch boundary bookkeeping such as logging epoch summaries.
-func (s *Service) handleEpochBoundary(ctx context.Context, postState state.BeaconState) error {
```

Comment:
The removal of this function should be in the description of the PR and most probably on the title as well.